src/main/java/core/basesyntax/FileWriterImpl.java

```diff
@@ -0,0 +1,13 @@
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) {
+        try (BufferedWriter writer = new BufferedWriter(new java.io.FileWriter(filePath))) {
+            writer.write(data);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Use try-with-resources to ensure that resources are closed properly even if an IOException is thrown.